bower.json

```diff
@@ -4,7 +4,7 @@
   "main": "angular-lock.js",
```

Comment:
You're missing a version update in this file as well

Reply:
Removed that field as per this comment https://github.com/bower/bower/issues/2389#issuecomment-259408378.